src/components/BulkEnrollmentPage/BulkEnrollment.scss

```diff
@@ -10,3 +10,14 @@
     padding-left: 0;
   }
 }
+
```

Comment:
For my own learning: I've previously followed conventions where things like .list-item would be placed as close to the relevant component as possible instead of on the parent.  Does doing it this way just help to make the component itself more reusable, as the styling would change when the component is used elsewhere?

Reply:
Oh, yeah, it should be moved. I threw the styles here out of laziness and planned to clean them up, but then completely forgot about it.